build/Publish.targets

```diff
@@ -5,6 +5,25 @@
   <Import Project="$(MSBuildThisFileDirectory)/publish/Badge.targets" />
   <Import Project="$(MSBuildThisFileDirectory)/publish/Checksum.targets" />
 
+  <ItemGroup>
+    <SdkInstallerFileItemGroup Include="$(SdkInstallerFile)" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <SdkDebianUploadUrl>$(DotnetBlobRootUrl)/$(Product)/$(FullNugetVersion)/%(SdkInstallerFileItemGroup.Filename)%(SdkInstallerFileItemGroup.Extension)</SdkDebianUploadUrl>
```

Comment:
Yea, these seem related to the ones above. I wonder if publishing to the debian repo isn't a separate concern. Perhaps this target should move to its own targets file with the associated properties?